web/php/DB/AllowedTags.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Wikidot\DB;
+
+use Ozone\Framework\Database\Criteria;
+use Wikidot\DB\AllowedTagsPeer;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * Object Model Class.
+ *
+ */
+class AllowedTags extends AllowedTagsBase
+{
+    public static function getAllowedTags($siteId): string {
+        $c = new Criteria();
+        $c->add("site_id", $siteId);
+        $dbTags = AllowedTagsPeer::instance()->selectByCriteria($c);
+        $allowedTags = '';
+        foreach ($dbTags as $dbTag) {
+            $allowedTags .= htmlspecialchars($dbTag->getTag()).' ';
+        }
+        return trim($allowedTags);
```

Comment:
Why is this using space-separated tags in a string instead of `array<string>`? In the uses I can see you immediately `explode()`, so I don't think it makes sense to store it this way.